delivery_dhl_parcel/models/delivery_carrier.py

```diff
@@ -202,7 +207,7 @@ def dhl_parcel_cancel_shipment(self, pickings):
                 % picking.carrier_tracking_ref
             )
 
-    def dhl_parcel_get_label(self, carrier_tracking_ref, label_format="pdf"):
+    def dhl_parcel_get_label(self, carrier_tracking_ref):
```

Comment:
Keep the method signature even if it doesn't nothing. We don't want to break people's stuff. Add a comment to deprecate in future versions.

Reply:
Ok, si quieres mantenemos el método con el parámetro label_format pero no se tendrá en cuenta porque en dhl_parcel_request se usará lo definido en el transportista.